core/shared/src/main/scala/zio/prelude/newtypes/package.scala

```diff
@@ -107,4 +107,51 @@ package object newtypes {
   object FailureOut extends NewtypeF
 
   type FailureOut[+A] = FailureOut.Type[A]
+
+  object Natural extends SubtypeSmart[Int](isGreaterThanEqualTo(0)) {
```

Comment:
A type for integers in Scala is simply called `Int`. So why don't we go with this convention and call this `Nat`?
Not a big deal, but the shorter name could nudge more people to use it (which they should!).

Reply:
I'm fine either way. I think this is what we have used in our examples in the documentation but have also seen lots of sources that use `Nat`. The other consideration here is I think right now unfortunately this is not a great user facing data type because the definitions of the arithmetic operators on `Int` shadow the extension methods we would like to define.